crates/router/src/connector/intuit/transformers.rs

```diff
@@ -0,0 +1,277 @@
+pub struct IntuitAuthUpdateRequest {
+    grant_type: IntuitAuthGrantTypes,
+    refresh_token: String,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum IntuitAuthGrantTypes {
+    RefreshToken,
+}
+
+impl TryFrom<&types::RefreshTokenRouterData> for IntuitAuthUpdateRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::RefreshTokenRouterData) -> Result<Self, Self::Error> {
+        Ok(Self {
+            grant_type: IntuitAuthGrantTypes::RefreshToken,
+            refresh_token: item.get_request_id()?,
+        })
+    }
+}
+
+#[derive(Default, Debug, Clone, Deserialize, PartialEq)]
+pub struct IntuitAuthUpdateResponse {
+    access_token: String,
+    expires_in: i64,
+    refresh_token: String,
+    x_refresh_token_expires_in: i64,
+}
+
+impl<F, T> TryFrom<types::ResponseRouterData<F, IntuitAuthUpdateResponse, T, types::AccessToken>>
+    for types::RouterData<F, T, types::AccessToken>
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: types::ResponseRouterData<F, IntuitAuthUpdateResponse, T, types::AccessToken>,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            response: Ok(types::AccessToken {
+                token: item.response.access_token,
+                expires: item.response.expires_in,
+            }),
+            ..item.data
+        })
+    }
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct IntuitPaymentsRequest {
+    amount: String,
+    currency: enums::Currency,
+    description: Option<String>,
+    context: IntuitPaymentsRequestContext,
+    card: Card,
+    capture: bool,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Card {
+    number: Secret<String, pii::CardNumber>,
+    exp_month: Secret<String>,
+    exp_year: Secret<String>,
+    cvc: Secret<String>,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct IntuitPaymentsRequestContext {
+    mobile: bool,
+    is_ecommerce: bool,
+}
+
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for IntuitPaymentsRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
+        match item.request.payment_method_data {
+            api::PaymentMethod::Card(ref ccard) => {
+                let submit_for_settlement = matches!(
+                    item.request.capture_method,
+                    Some(enums::CaptureMethod::Automatic) | None
+                );
+                Ok(Self {
+                    amount: item.request.amount.to_string(),
+                    currency: item.request.currency,
+                    context: IntuitPaymentsRequestContext {
+                        mobile: item.request.browser_info.clone().map_or(true, |_| false),
```

Comment:
what is the significance of this field, can you add comments

Reply:
Added comments in `struct IntuitPaymentsRequestContext` for the same